### DISCUSSION TITLE
Radiomaster tx12 DSM

### DISCUSSION THREAD

**Discussion by makaidi39** [pascallanger/DIY-Multiprotocol-TX-Module#549] [Q&A]
Hey
Pretty new to this stuff, and just a quick question.
Does Radiomaster TX12 support DSM protocol? Doesn't seem to be able to choose it on the Radio. can see the TX16 can, and they both use the same chip, so I might be doing something wrong ?

**Comment by benlye** [comment]
No, the TX12 does not support DSM.  It does not have the 4-in-1 MULTI-Module like the TX16S.
